libyul/backends/evm/EVMDialect.cpp

```diff
@@ -222,3 +213,77 @@ SideEffects EVMDialect::sideEffectsOfInstruction(evmasm::Instruction _instructio
 		evmasm::SemanticInformation::invalidatesMemory(_instruction)
 	};
 }
+
+EVMDialectTyped::EVMDialectTyped(langutil::EVMVersion _evmVersion, bool _objectAccess):
+	EVMDialect(_evmVersion, _objectAccess)
+{
+	defaultType = "u256"_yulstring;
+	boolType = "bool"_yulstring;
+	types = {defaultType, boolType};
+
+	m_functions["lt"_yulstring].returns = {"bool"_yulstring};
```

Comment:
Does it make sense to test them in one single place? Right now, it seems we do not test all of them and tests are distributed over several testing stages.

Reply:
I think we can only properly test them when type checking is in place.

Reply:
Ok, fine with that.